packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -279,6 +286,7 @@ export interface IContainerRuntimeOptions {
      * 3. "bypass" will skip the check entirely. This is not recommended.
      */
     loadSequenceNumberVerification?: "close" | "log" | "bypass";
+    useDataStoreAliasing?: boolean;
```

Comment:
Enabling this currently breaks the test flow due to how the test containers are created:

```
  Error: Trying to submit message while detached!
      at Object.assert (/home/andrei/work/clone6/FluidFramework/node_modules/@fluidframework/common-utils/src/assert.ts:15:16)
      at DataStore.trySetAlias (/home/andrei/work/clone6/FluidFramework/packages/runtime/container-runtime/src/dataStore.ts:28:9)
      at ContainerRuntime.createRootDataStore (/home/andrei/work/clone6/FluidFramework/packages/runtime/container-runtime/src/containerRuntime.ts:1579:44)
      at Object.instantiateFirstTime (/home/andrei/work/clone6/FluidFramework/packages/test/test-utils/src/testContainerRuntimeFactory.ts:30:13)
```

Investigating